bigbluebutton-html5/imports/ui/components/user-list/user-list-content/user-participants/user-list-participants/list-item/component.tsx

```diff
@@ -181,11 +181,15 @@ const UserListItem: React.FC<UserListItemProps> = ({ user, lockSettings }) => {
     } return '';
   };
 
+  const isVoiceOnly = user.isDialIn;
+
   const iconUser = getIconUser();
+  const iconVoiceOnlyUser = (<Icon iconName="volume_level_2" />);
+  const userIcon = isVoiceOnly ? iconVoiceOnlyUser : iconUser;
```

Comment:
Turned out that only dial-in users are considered `voiceOnly`.
So I suggest to name it `isDialIn` instead of `isVoiceOnly`. It will keep things simpler.